Svc/SystemResources/MemUtilPortAi.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../Autocoders/Python/schema/ISF/interface_schema.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+
+<interface name="MemUtil" namespace="Svc">
+    <comment>
+        System Memory Utilization
+    </comment>
+    <args>
+        <arg name="memUsed" type="F32">
```

Comment:
Type F32, is this right? Trying to fit a U64 into a F32?

Reply:
Unused port removed.